operator/internal/handlers/lokistack_create_or_update.go

```diff
@@ -61,7 +61,7 @@ func CreateOrUpdateLokiStack(ctx context.Context, req ctrl.Request, k k8s.Client
 		return kverrors.Wrap(err, "failed to lookup lokistack s3 secret", "name", key)
```

Comment:
Mention of `s3`: Should we base this on the `type` of secret instead?

Reply:
I'll leave off `type` for now as this error is more indicative that we can't find the secret and not that we can't find the type of secret.